addons/hr_expense/models/hr_expense.py

```diff
@@ -499,21 +499,19 @@ def _onchange_employee_id(self):
         self.address_id = self.employee_id.address_home_id
         self.department_id = self.employee_id.department_id
 
-    @api.multi
-    def check_consistency(self):
-        if any(sheet.employee_id != self[0].employee_id for sheet in self):
-            raise UserError(_("Expenses must belong to the same Employee."))
-
-        expense_lines = self.mapped('expense_line_ids')
-        if expense_lines and any(expense.payment_mode != expense_lines[0].payment_mode for expense in expense_lines):
-            raise UserError(_("Expenses must have been paid by the same entity (Company or employee)"))
-
-    @api.one
     @api.constrains('expense_line_ids')
```

Comment:
@jem-odoo Are you sure the method will be triggered if a payment_mode is modified on an expense_line ?